### PR TITLE
[staging] Adding barebones step-by-step documentation 

### PR DESCRIPTION
**Pull request by dantaeyoung**
This is very barebones - feedback much welcome!

### CODE REVIEW

File: SETUP.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+#### praw.ini
+- `cp praw.ini.example praw.ini`
+- Create Reddit application on reddit.com; fill in `oauth_client_id`, `oauth_client_secret` with application values 
+  - (Further documentation in `docker/README.md`)
+
+#### config/production.json
+- `cp config/development.json.example config/production.json`
+  - (NOTE: filename probably changes according to environment)  
+- Fill out `host`/`database`/`user`/`password`
+  - (client_id, client_secret, redirect_uri are most likely not used)
```
Comment (wllrd) [inline SETUP.md:27]:
```suggestion
  - (client_id, client_secret, redirect_uri are not used)
```

confirmed!

File: SETUP.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+- `CS_ENV=production python set_up_auth.py`
+  - Navigate to generated reddit URL in browser
+  - Enter code from redirect_url into python script
+- `config/access_information_production.pickle` should be created
+
+## Services
+
+### Redis-server 
+ - Should already be running
```
Comment (wllrd) [inline SETUP.md:55]:
```suggestion
 - Should already be running (installed as a system service)
```

File: SETUP.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+- praw-multiprocess
+
+
+## Schedule Jobs
+
+To schedule jobs (such as fetching modactions, new posts, comments)
+
+- Load environment variables
+  - `source config/environment_variables.sh`
+- Patch praw
```
Comment (wllrd) [inline SETUP.md:81]:
```suggestion
- Patch praw (first run only)
```

**Review by wllrd — APPROVED** [review]
good distillation of our notes! i suggested some confirmations of your assumptions in a few places.

**Comment by dantaeyoung** [comment]
thanks for giving it a once-over!

**Pull request merged**